tabs/modes.js

```diff
@@ -11,10 +11,6 @@ TABS.modes.initialize = function (callback) {
         googleAnalytics.sendAppView('Modes');
     }
 
-    function get_box_data() {
```

Comment:
To be honest I would leave this message in place. periodicStatusUpdater can be postponed by serial port balancing and it's not guaranteed that active modes will be loaded when TAB is entered

Reply:
MSP_BOX was not implemented at all - it was reused as MSP_ACTIVEBOXES.

Reply:
I'll add it back